tar5/vinaigrette.py

```diff
@@ -0,0 +1,36 @@
+from random import randrange
+from datetime import timedelta
+from datetime import datetime
+import calendar
+
+
+def random_date(start_date, end_date):
+    """
+    This function get two datetime objects: start and end.
+    return a random datetime between two datetime objects.
+    """
+    delta = end_date - start_date
+    int_delta = (delta.days * 24 * 60 * 60)
+    random_second = randrange(int_delta)
+    return start + timedelta(seconds=random_second)
+
+
+def if_monday():
+    """
+    The function picks up two dates from the user and sends to the function for a random date between them.
+     For a date that is Monday a message will be printed
+    """
+    start_date = datetime.strptime(input("enter date 1: "), '%m-%d-%Y')
+    end_date = datetime.strptime(input("enter date 2: "), '%m-%d-%Y')
+    my_date = random_date(start_date, end_date)
+    print(my_date)
+    if calendar.day_name[my_date.weekday()] == 'Monday':

```

Comment:
We prefer that each function will do only one job, so it would be better if the action of checking whether this is a monday will be divided into another function.

Reply:
i fix